.trunk/trunk.yaml

```diff
@@ -61,6 +61,7 @@ lint:
         - lib*
         - resources*
         - tests/ef_tests/test_data
+        - .katana/messaging_config.json
```

Comment:
why do we need to exclude it?

Reply:
it uses a private key and trunk detects it